NewRelicBrowser.php

```diff
@@ -1,392 +1,456 @@
-					)
+					'id'      => 'rtp_relic_page_about_tab',
+					'title'   => esc_html__( 'New Relic Browser', 'rt-new-relic' ),
+					'content' => '<p>' . esc_html__( 'New Relic Browser provides deep visibility and actionable insights into real users experiences on your website. With standard page load timing (sometimes referred to as real user monitoring or RUM), New Relic measures the overall time to load the entire webpage. However, New Relic Browser goes beyond RUM to also help you monitor the performance of individual sessions, AJAX requests, and JavaScript errors—extending the monitoring throughout the entire life cycle of the page.', 'rt-new-relic' ) . '</p>',
+				)
 			);
 		}
 
 		/**
 		 * Add settings page
-		 * @return null
+		 *
+		 * @return void
 		 */
-		function new_relic_options()
-		{
+		public function new_relic_options() {
 			include 'admin/new-relic-admin.php';
 		}
 
 		/**
-		 *  Enqueue scripts and styles
-		 * @return null
+		 * Enqueue scripts and styles
+		 *
+		 * @return void
 		 */
-		function rtp_relic_load_js()
-		{
-			wp_enqueue_script( 'rtp_relic_js', plugins_url( '/js/rtp-relic.js', __FILE__ ), array( 'jquery' ) );
+		public function rtp_relic_load_js() {
+			wp_enqueue_script( 'rtp_relic_js', plugins_url( '/js/rtp-relic.js', __FILE__ ), array( 'jquery' ), RTP_RESOURCE_VERSION, true );
 			wp_enqueue_script( 'jquery-ui-dialog' );
-			wp_enqueue_style( 'rtp_relic_css', plugins_url( '/css/rtp-new-relic.css', __FILE__ ) );
-			wp_enqueue_style( 'rtp_relic_ui_css', plugins_url( '/css/jquery-ui.css', __FILE__ ) );
+
+			wp_enqueue_style( 'rtp_relic_css', plugins_url( '/css/rtp-new-relic.css', __FILE__ ), array(), RTP_RESOURCE_VERSION );
+			wp_enqueue_style( 'rtp_relic_ui_css', plugins_url( '/css/jquery-ui.css', __FILE__ ), array(), RTP_RESOURCE_VERSION );
 		}
 
 		/**
-		 * Validate user form
-		 * @param array $relic_user_data
+		 * Validate user form.
+		 *
+		 * @param array $relic_user_data Relic User Data.
+		 *
 		 * @return array
 		 */
-		function rtp_relic_validate_form( $relic_user_data )
-		{
-
-			$relic_valid = true;
+		public function rtp_relic_validate_form( $relic_user_data ) {
+			$relic_valid         = true;
 			$relic_error_message = '';
+
 			if ( isset( $relic_user_data['rtp-relic-form-name'] ) ) {
 				$form_name = sanitize_text_field( $relic_user_data['rtp-relic-form-name'] );
 			}
 
-			if ( 'rtp-get-browser' == $form_name ) {
+			if ( 'rtp-get-browser' === $form_name ) {
 				/* get browser list form */
-				if ( ( '' == $relic_user_data['rtp-user-api-key'] ) ) {
-					$relic_valid = false;
-					$relic_error_message = __( 'All fields are required.', 'rt-new-relic' );
+				if ( ( '' === $relic_user_data['rtp-user-api-key'] ) ) {
```

Comment:
Use `empty()` instead of comparing empty string.

Reply:
i solved this issue. please review changes.